SOOP/SOOP_ASF_SST_XBT/SOOP_BOM_ASF_SST.py

```diff
@@ -1,15 +1,15 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
+
+from imos_logging import IMOSLogging
+from lftp_sync import LFTPSync
+from tendo import singleton
+import argparse
+import fnmatch
```

Comment:
While you're re-ordering all these imports, it might be nice to follow [PEP 8](https://www.python.org/dev/peps/pep-0008/#id23) (though I can't say I've been doing this consistently):
> Imports should be grouped in the following order:
>   1. standard library imports
>   2. related third party imports
>   3. local application/library specific imports
>
> You should put a blank line between each group of imports.